pages/_document.tsx

```diff
@@ -10,6 +10,12 @@ class MyDocument extends Document {
     return (
       <Html lang="pt" className="antialiased p-0 mx-0 outline-none box-border">
         <Head>
+          <link
+            rel="preload"
+            href="/fonts/montserrat-v15-latin-600.woff2"
+            as="font"
+            crossorigin="anonymous"
```

Comment:
The attribute should be camel case. Refer to: https://en.reactjs.org/docs/dom-elements.html#all-supported-html-attributes

![image](https://user-images.githubusercontent.com/48022589/99805148-d3f31780-2b1a-11eb-8338-a946f3b976d1.png)